.github/workflows/e2e_tests.yml

```diff
@@ -0,0 +1,74 @@
+name: End-to-End Tests
+on:
+  push:
+    branches:
+      - main
+      - master
+  pull_request:
+  # schedule:
+  #   - cron: "0 8 * * *"
+jobs:
+  e2e-test:
+    name: E2E Test Python ${{ matrix.py }} - Postgis ${{ matrix.postgis }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        py:
+          - "3.10"
+          # - "3.9"
```

Comment:
Are we keeping these around for a reason?

Reply:
Posterity? JK. I'll remove them.

Reply:
Done. I'd like to keep the PostGIS versions for now though. I plan to get it working for 3.2 soon.